pages/en/learn/test-runner/using-test-runner.md

```diff
@@ -0,0 +1,300 @@
+# Using Node.js's test runner
+
+```
+example/
+  ├ …
+  ├ src/
+    ├ app/…
+    └ sw/…
+  └ test/
+    ├ globals/
+      ├ …
+      ├ IndexedDb.js
+      └ ServiceWorkerGlobalScope.js
+    ├ setup.mjs
+    ├ setup.units.mjs
+    └ setup.ui.mjs
+```
+
+
+```json
+{
+  "name": "example",
+  "scripts": {
+    "test": "concurrently --kill-others-on-fail --prefix none npm:test:*",
+    "test:sw": "node --import ./test/setup.sw.mjs --test './src/sw/**/*.spec.*'",
+    "test:units": "node --import ./test/setup.units.mjs --test './src/app/**/*.spec.*'",
+    "test:ui": "node --import ./test/setup.ui.mjs --test './src/app/**/*.test.*'"
+  },
+} 
+```
```

Comment:
```suggestion
Node.js has a built-in test runner that is very powerful and flexible. This guide will show you how to set up a test runner using Node.js's built-in test runner.

"""text displayName="Architecture overview"
example/
  ├ …
  ├ src/
    ├ app/…
    └ sw/…
  └ test/
    ├ globals/
      ├ …
      ├ IndexedDb.js
      └ ServiceWorkerGlobalScope.js
    ├ setup.mjs
    ├ setup.units.mjs
    └ setup.ui.mjs
"""

"""bash displayName="Install dependencies"
npm i -y
npm install --save-dev concurrently
"""

"""json displayName="package.json"
{
  "name": "example",
  "scripts": {
    "test": "concurrently --kill-others-on-fail --prefix none npm:test:*",
    "test:sw": "node --import ./test/setup.sw.mjs --test './src/sw/**/*.spec.*'",
    "test:units": "node --import ./test/setup.units.mjs --test './src/app/**/*.spec.*'",
    "test:ui": "node --import ./test/setup.ui.mjs --test './src/app/**/*.test.*'"
  },
}
"""
```

<img width="1552" alt="Capture d’écran 2024-06-02 à 18 08 19" src="https://github.com/nodejs/nodejs.org/assets/97875033/bba39262-709a-43a3-a5a9-33ce19e62191">

> [!NOTE]
> you I'll need to replace " by ` it's just because GitHub didn't like code snippet on suggestions 😄